src/topology/homotopy/product.lean

```diff
@@ -0,0 +1,83 @@
+/-
+Copyright (c) 2021 Praneeth Kolichala. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Praneeth Kolichala
+-/
+import topology.homotopy.basic
+import topology.constructions
+
+/-!
+# Product of homotopies
+
+In this file, we introduce definitions for the product of
+homotopies. We show that the products of relative homotopies
+are still relative homotopies.
+
+## Definitions
+- `homotopy.pi homotopies`: Let f and g be a family of functions
```

Comment:
```suggestion
- `continuous_map.homotopy.pi homotopies`: Let f and g be a family of functions
```
I think you will need the fully qualified name here, so `continuous_map.homotopy.pi` for doc-gen to generate the correct link.